tests/unit/admin/test-class-wc-rest-payments-webhook.php

```diff
@@ -173,15 +173,21 @@ public function test_valid_failed_refund_update_webhook() {
 
 		$mock_order = $this->getMockBuilder( WC_Order::class )
 			->disableOriginalConstructor()
-			->setMethods( [ 'add_order_note' ] )
+			->setMethods( [ 'add_order_note', 'get_meta' ] )
 			->getMock();
 
+		$mock_order
+			->expects( $this->once() )
+			->method( 'get_meta' )
+			->with( '_wcpay_original_order_currency' )
+			->willReturn( 'EUR' );
+
 		$mock_order
 			->expects( $this->once() )
 			->method( 'add_order_note' )
 			->with(
 				$this->matchesRegularExpression(
-					'~^A refund of <span class="woocommerce-Price-amount amount">(<bdi>)?<span class="woocommerce-Price-currencySymbol">&pound;</span>9.99(</bdi>)?</span> was <strong>unsuccessful</strong> using WooCommerce Payments \(<code>test_refund_id</code>\).$~'
+					'~^A refund of <span class="woocommerce-Price-amount amount">(<bdi>)?<span class="woocommerce-Price-currencySymbol">&euro;</span>9.99(</bdi>)?</span> was <strong>unsuccessful</strong> using WooCommerce Payments \(<code>test_refund_id</code>\).$~'
```

Comment:
🤔 Let's create a new similar test case for that. This applies to all other edited tests as well.

Reply:
Good point.

Reply:
Done.